home.md

```diff
@@ -287,8 +287,8 @@
     <p><h4>Complete under-the-hood visualization of the your system's behavior</h4></p>
```

Comment:
@johnsimons - I'd already merged this PR. Feel free to fix this and push to master. Btw, thanks, I wasn't paying attention to anything else besides SM :)